core/src/impl/Kokkos_SimpleTaskScheduler.hpp

```diff
@@ -394,6 +403,20 @@ class SimpleTaskScheduler
     return this->team_scheduler_info_storage::no_unique_address_data_member();
   }
 
+  //----------------------------------------------------------------------------
+
+  // For backwards compatibility purposes only
+  KOKKOS_INLINE_FUNCTION
+  KOKKOS_DEPRECATED_FUNCTION
```

Comment:
What is the right sequence?  1st inline specifier, then deprecated attribute?

Reply:
Yeah, these should be switched